src/_lib/sass/src/_variables.scss

```diff
@@ -90,7 +91,7 @@ $link-hover-decoration: underline !default;
 
 //== Typography
 //
-$font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, sans-serif !default;
+$font-family-sans-serif:  "ProximaNova", "Helvetica Neue", Helvetica, Arial, sans-serif !default;
```

Comment:
Do we know what we're doing wit the fonts that Analytics require?

Reply:
For now we can handle them with local font-family overrides